Functions/xASL_adm_GetDeprecatedFields.m

```diff
@@ -85,11 +86,14 @@
     'bSkipIfNoM0'                   'settings'  'bSkipIfNoM0'                        '';...
     'bSkipIfNoFlair'                'settings'  'bSkipIfNoFlair'                     '';...
     'bRunModule_DARTEL'             'modules'   'bRunDARTEL'                         '';...
+    'bRunDARTEL'                    'modules'   'bRunDARTEL'                         '';...
     'bRunModule_LongReg'            'modules'   'bRunLongReg'                        '';...
+    'bRunLongReg'                   'modules'   'bRunLongReg'                        '';...
     'ForceInclusionList'            'dataset'   'ForceInclusionList'                 '';...
     'bAutomaticallyDetectFS'        'external'  'bAutomaticallyDetectFS'             '';...
     'MakeNIfTI4DICOM'               'settings'  'MakeNIfTI4DICOM'                    '';...
     'SegmentSPM12'                  'modules'   'structural'                         'bSegmentSPM12';...
```

Comment:
Wasn't it Segment_SPM12?

Function `Functions/xASL_adm_GetDeprecatedFields.m` has it together, function `Functions/xASL_init_DefineDataDependentSettings.m` has it with the underscore...

Reply:
I think this is the easiest fix: e1ae03f 
I also updated the table 👍 
This way we use the same script to check the backwards compatibility everywhere 👍